DataFormats/METReco/src/MET.cc

```diff
@@ -68,6 +68,24 @@ MET::MET( double sumet_, const std::vector<CorrMETData>& corr_,
     }
 }
 
+// Copy constructor
+//____________________________________________________________________________||
+MET::MET( const MET& met ): 
```

Comment:
@mariadalfonso 
this code is still hare after the last update.
I note that 80X version does not have it